specs/AppNotifications/AppNotifications-spec.md

```diff
@@ -144,17 +144,39 @@ int main()
 For Unpackaged applications, the developer can Register using a custom Display Name and Icon.
 WinAppSDK will register the application and display these assets when an App Notification is received.
 The developer should provide both the assets or not provide them at all. Icon provided by the developer
-should be a valid supported format. The API supports the formats - png, bmp, jpg. The icon
+should be a valid supported format. The API supports the formats - png, bmp, jpg, ico. The icon
 should reside on the local machine only otherwise the API throws an exception. For Packaged applications,
-this API is not applicable and will throw an exception.
+this API is not applicable and will throw an exception. Below are some examples of usage:
 
 ```cpp
 int main()
 {
     auto manager = winrt::AppNotificationManager::Default();
 
     std::wstring iconFilepath{ std::filesystem::current_path() / "icon.ico" };
-    manager.Register(L"AppNotifications", winrt::Windows::Foundation::Uri {iconFilepath});
+    winrt::hstring displayName{ L"AppNotifications" };
+
+    manager.Register(displayName, winrt::Windows::Foundation::Uri {iconFilepath});
+
+    // other app init and then message loop here
+
+    // Call Unregister() before exiting main so that subsequent invocations will launch a new process
+    manager.Unregister();
+    return 0;
+}
+```
+
+```cpp
+int main()
+{
+    auto manager = winrt::AppNotificationManager::Default();
+
+    std::wstring iconFilepath{ std::filesystem::current_path() / "icon.ico" };
+
+    std::wstring displayName{};
+    wil::GetModuleFileNameExW(GetCurrentProcess(), nullptr, displayName);
```

Comment:
The thing that doesn't make me comfortable about this is that we pass a filename as the display name (the dev is free to pass whatever value they want, but the point of this API is to let devs to pass a pretty display name to be shown on the app notifications).

Reply:
these are just examples to show them they could do something like this but yeah the developers are free to pass whatever pretty value they want and we should adhere to it and we are. :)

Reply:
Why don't you declare a const called c_displayName and assign it to something like "DemoToastApp"? Then reuse that here.